### PR TITLE
[PM-287] Add aria-label to account button

### PR DESCRIPTION
**Pull request by jprusik**
## Type of change

```
- [X] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Add aria-label to the account dropdown button so that voiceover tools read out the text:
"Account: Logged in as [account name]"

## Code changes

Added aria-label to button and corresponding i18n translation catalog entry

## Screencaps

https://user-images.githubusercontent.com/1556494/235766036-e6b80a36-c5f4-4647-ad4f-79a8f33a8434.mp4

https://user-images.githubusercontent.com/1556494/235766075-bee6657f-370a-450b-b7e3-fef835d1d225.mp4

### CODE REVIEW

**Review by differsthecat — DISMISSED** [review]
This looks great! There is a translation key with "Logged in as" as the message, but it is missing the "Account:" at the beginning, so we do need to add this one.

File: apps/web/src/app/layouts/navbar.component.html [hunk 1]
```diff
@@ -46,6 +46,7 @@
         <button
           [bitMenuTriggerFor]="accountMenu"
           class="tw-border-0 tw-bg-transparent tw-text-alt2 tw-opacity-70 hover:tw-opacity-90"
+          attr.aria-label="{{ 'accountLoggedInAsEmail' | i18n : name }}"
```
Comment (differsthecat) [inline apps/web/src/app/layouts/navbar.component.html:49]: This translation key implies that the email is passed in, but it looks like the account name is passed instead; should it be the email?

I can see someone having multiple accounts with the same name, but different emails.

If using the name is intended though, I'd change the translation key to "accountLoggedInAsName".

**Review by differsthecat — COMMENTED** [review]
Sorry, one small thing I noticed right after I approved!

File: apps/web/src/app/layouts/navbar.component.html (hunk 1 above)
Comment (jprusik) [reply apps/web/src/app/layouts/navbar.component.html:49]: yup, that makes sense - an oversight on my part copying from prior art 👍

**Comment by jprusik** [comment]
> This looks great! There is a translation key with "Logged in as" as the message, but it is missing the "Account:" at the beginning, so we do need to add this one.

Indeed; I wanted to avoid potential string concatenation problems with translation

**Review by differsthecat — APPROVED** [review]
Thank you @jprusik !

**Pull request merged**